routes/web.php

```diff
@@ -30,10 +32,11 @@
 
 Route::get('/blog')->name('blog');
 Route::get('/about')->name('about');
-Route::view('/contacts', 'contacts')->name('contacts');
-
+Route::view('/contacts', 'contacts')->middleware(\App\Http\Middleware\RequestLoggingMiddleware::class)->name('contacts');
```

Comment:
middleware should be applied for all endpoints in you case, you can use group or the register it in Kernel.